File: ejercicios-Python/ejercicio_5.py
def ejercicio_5(peso, estatura):
    """La funcion toma el peso y la estatura de una persona
       y calcula su IMC

    Args:
        peso (number): Peso de la persona en Kg
        estatura (number): Estatura de la persona en m
    """
    imc = peso/estatura
    print(f'Tu indice de masa corporal (IMC) es {imc}')

File: ejercicios-TypeScript/ejercicio_5.ts
const ejercicio_5 = (peso: number, estatura: number): void => {
  /**
   * La funcion toma el peso y la estatura de una persona
   * calcula su IMC.
   *
   * Args:
   *  peso (number): Peso de la persona en Kg
   *  estatura (number): El estatura de la persona en m
   */

  let imc: number = peso / estatura;
  console.log(`Tu indice de masa corporal es ${imc}`);
};
